apps/ledger-live-desktop/tests/specs/speculos/manage.ledgersync.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test } from "../../fixtures/common";
+import { addTmsLink } from "tests/utils/allureUtils";
+import { getDescription } from "../../utils/customJsonReporter";
+import { AppInfos } from "tests/enum/AppInfos";
+
+// Use a single value instead of an array
+const app: AppInfos = AppInfos.LS;
+
+test.describe(`[${app.name}] Sync Accounts`, () => {
+  test.use({
+    userdata: "ledgerSync",
+    speculosApp: app,
+  });
+
+  test("Synchronize one instance then delete the backup", async ({ app, page }) => {
+    const description = getDescription(test.info().annotations).split(", ");
+    await addTmsLink(description);
+
+    // Step 1: Validate the presence of the sync entry point
```

Comment:
Remove com

Reply:
to me those comments are fine, helps to understand what is happening in the test